gavel/templates/begin.html

```diff
@@ -14,10 +14,11 @@ <h1>Start</h1>
   <div class="info">
     <form action="{{ url_for('begin') }}" method="post">
       <input type="submit" name="action" value="Skip" id="vote-skip" class="button-full">
+      <input type="submit" name="action" value="Absent" id="vote-absent" class="button-full button-gap">
```

Comment:
You can delete this change so that it will not give us a conflict -- Michael's PR #9 should be adding that.